keyboards/pabile/p20/ver2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 Pabile
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    DBLZERO = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case DBLZERO:
+        if (record->event.pressed) {
+            SEND_STRING("00");
+        } else {
+        }
```

Comment:
Honestly, I'd do this: 
```suggestion
        if (record->event.pressed) {
            tap_code(KC_P0);
            register_code(KC_P0);
        } else {
            unregister_code(KC_P0);
        }
```

This uses the correct zero, and will allow you to repeate it, if you want. 

Otherwise, this:
```suggestion
        if (record->event.pressed) {
            tap_code(KC_P0);
            tap_code(KC_P0);
        }
```

Reply:
wow! thanks! yes, you are correct. i like either approach but i'd go with the later for it contains lines i am more familiar with :)